impala/__init__.py

```diff
@@ -15,6 +15,20 @@
 
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 
+
+class SQLAlchemy(FlaskSQLAlchemy):
+    """
+    A hack because flask-sqlalchemy hasn't fixed this yet:
+    https://github.com/mitsuhiko/flask-sqlalchemy/issues/166
```

Comment:
Interesting. I'd just make sure to remove this once the open PR they have to fix this is closed.